tests/python3_test_files.txt

```diff
@@ -24,6 +24,7 @@ bgp/test_bgp_session_flap.py
 bgp/test_bgp_slb.py
 bgp/test_bgp_speaker.py
 bgp/test_bgp_update_timer.py
+bgp/test_bgp_dual_asn.py
```

Comment:
Do we need to add the test in https://github.com/sonic-net/sonic-mgmt/blob/master/.azure-pipelines/pr_test_scripts.yaml